scripts/ironic.profile

```diff
@@ -0,0 +1,4 @@
+REQUESTS_CA_BUNDLE=/proc/1/root/certs/ca/ironic-inspector/ca.crt
+OS_AUTH_TYPE=none
```

Comment:
It's not necessarily none, in most cases you'll have authentication.

Reply:
Of course, but I think it is a reasonable default for an example of how to use the image as debug container.
You may override it using `--env` on the kubectl call.

I added some documentation on using `--env` to the README.md file in this fixup-commit: https://github.com/metal3-io/ironic-client/pull/16/commits/954e59897f563551a46732eeb4d2b623fc61db3d

Would this suffice to resolve your concern?